lib/scrabble/scrabble.rb

```diff
@@ -9,8 +9,9 @@ class Scrabble
       8 => ["j", "x"],
       10 => ["q", "z"]
     }
+
     def self.score(word)
-      if word[/^[a-zA-Z]+/] != word
+      if (word[/^[a-zA-Z]+/] != word) || !(Dictionary.valid_word?(word)) #Check if it only has letters AND is a valid word
         return "That is not a valid word."
       elsif word.length > 7
         return "Cannot have more than seven letters."
```

Comment:
this comment is for line 28: yes, you can use `.each_char` :) Then you wouldn't need to create `word_array` at all -- just check `word.length` and score it using `word.each_char`.